### ISSUE TITLE
Having problem in rocky linux 8.9

### ISSUE THREAD

**Issue by ZeeRoon** [nitmir/policyd-rate-limit#26]
Hello,

I have been trying to install policyd-rate-limit in Rocky Linux 8.9 (Green Obsidian) with python 3.6 and pip 21.3.1.
first of all, after installation with pip install policyd-rate-limit, policyd-rate-limit doesn't start. there are missing file paths.
for example,  
/etc/init.d/policyd-rate-limit: line 29: /lib/lsb/init-functions: No such file or directory
policyd-rate-limit is not running ... failed! ,when, service policyd-rate-limit status is called
tried changing service policyd-rate-limit status with snapd's init-functions (/var/lib/snapd/snap/...) which has the same variables as called by /var/lib/snapd/snap/

and kindly add documentation links as I couldn't find any of integration with postfix. 
Thank you.

**Comment by nitmir** [comment]
Hello,

Your system is using systemd as an init system and so you should using the [systemd unit](https://github.com/nitmir/policyd-rate-limit/blob/master/init/policyd-rate-limit.service) provided in the source tree. The sysvinit script is for the sysvinit init system and won't work if sysvinit is not used.

I do not get why you're trying to use snap in this context.

Postfix configuration is handled in the `Postfix settings` section of the README

Postfix documentation about policy servers is available there https://www.postfix.org/SMTPD_POLICY_README.html

**Comment by fictitiousexistence** [comment]
Yeah, seems to work perfectly fine on Rocky Linux 9.4 with systemd.
